lib/bank_account/savings_account.rb

```diff
@@ -0,0 +1,57 @@
+module BankAccount
+
+  class SavingsAccount < Account
+    attr_reader :id, :balance
+
+    def initialize(id, initial_balance)
+      @id = id
+      @balance = initial_balance
+      # again, made initial balance equal to instance variable balance, which actually turns out to be the current balance
+
+      #this raises an ArgumentError to make sure that the initial balance can't be lower than 10
+      unless @balance >= 10
+        raise ArgumentError.new("You can't start out with a balance below $10. Add moar monies.")
+      end
+
+    end
+
+    def withdraw(amount)
+
+      # I made a new variable to represent the transaction fee added to the amount of withdrawal
```

Comment:
Good use of a comment!  Otherwise, you could go without the comment and add a new variable called `fee` and then `amount_with_fee = amount + fee`, but this way is nice and readable too. :)